src/app/service-box/service-box.component.scss

```diff
@@ -2,13 +2,21 @@
 	position: relative;
 }
 
+#side-menu{
+  width: 40px;
+  float: right;
+  position: absolute;
+  right: 30px;
+  z-index: 2;
+}
+
 .box {
 	border: 1px solid #ccc;
 	box-shadow: 0 2px 10px rgba(0,0,0,.2);
 	padding: 30px;
 	position: absolute;
-	right: 0;
-	top: 45px;
+	right: 0px;
```

Comment:
Also, this change shouldn't be there. We should omit the unit with zero values in css.

Reply:
Are you referring to remove 'right' parameter from css which I have included ?

Reply:
No, just remove the unit.

Reply:
Okay, thanks. I'm making the changes.

Reply:
@singhpratyush there are some other parameters in the given scss file with 0 value and px unit shall I remove them also?